CONTRIBUTING.md

```diff
@@ -153,6 +153,23 @@ rather than conform to specific API names or argument patterns in the spec.
 
 For a deeper discussion, see: https://github.com/open-telemetry/opentelemetry-specification/issues/165
 
+## Running Tests Locally
+
+1. Go to your Contrib repo directory. `cd ~/git/opentelemetry-python-contrib`.
+2. Create a virtual env in your Contrib repo directory. `python3 -m venv my_test_venv`.
+3. Activate your virtual env. `source my_test_venv/bin/activate`.
```

Comment:
this only works for bash but may be that's good enough for the example.

Reply:
For virtualenv steps, should we link to virtualenv docs instead? Step 2. could be `create and activate virtualenv. [how to setup virtualsenv](link-to-docs)` ?